docs/use_cases/recommender_systems.md

```diff
@@ -1,15 +1,111 @@
+
+
+### Recommender Systems as Graphs
+
+Matrix factorization is a common collaborative filtering approach. It represents users and items as vectors in a shared latent space. The inner product of a user and item vector estimates the rating or interaction strength. We can view this process through a graph perspective. Users and items become graph nodes. Predicted ratings are edge weights between them. Recommendations identify the most likely new connections for a user. With this graph view, we can easily inject additional nodes as needed. For example, a "genre" node that links related items. This couples similar items which improves recommendations.
+
+(We deliberately took a detour and observed the problem via graphs. Note that this approach is known as ‘joint matrix factorization’ in the literature).
+
+Once the problem is understood as inherently a graph problem, it is easy to see how additional information can be added. What we want to achieve is to somehow let the algorithm know about the new links, which would help group similar items or users together. In other words, we want to ‘inject’ new nodes, which would link the nodes belonging to the same group. How could this be achieved? Have a look at illustration below:
+
+![](assets/use_cases/recommender_systems/dummy_nodes.jpg)
+
+There are three users, U1, U2 and U3, and four items, I1, I2, I3, I4. The user U1 has interacted with items I1 and I3. Now, there is a Dummy User, who links the items that have the same color (I1, I3, I4). 
+The dummy user couples similar items, helping the model identify related content. This increases the chances of item I4 to be recommended to user U1.
+
+**Adaptation**
+
+As discussed above, we only need to add dummy data: in case of only item side information available, add dummy users; in case of only user side information available, add dummy items; in case both the user and the item information is available, add both the dummy users and the dummy items. Obviously, the dummy nodes should not be part of recommendations; these nodes are only there to help ‘inject’ some ‘commonality’ of the nodes belonging to a certain group.
+
+Beside MF, the same approach can be used with [EASE](https://arxiv.org/abs/1905.03375), for example, or with an explicitly graph-based approach for recommendations such as [PageRank](https://scikit-network.readthedocs.io/en/latest/use_cases/recommendation.html). In fact, with PageRank, the walks would include the dummy nodes.
+
+One question remains: which weights (ratings) to put for the dummy user interactions? We suggest you first start with low weights, and see how the quality of the recommendations changes.
```

Comment:
Previous review comments - not yet addressed:

This is a short article. Try to avoid repeating the same phrases (e.g., "one question").
It's not accurate to say "one question remains" unless you set up a problem with fixed parameters and a set of questions to answer ahead of time.
"which weights (ratings) to put for the dummy user interactions?" = not grammatically a question. This section could be called "How to decide on weights (ratings) for dummy user interactions" and the question: "How should dummy user interactions be weighted (rated)?"

Q: Is the correct term "low weights" or "low weightings"?

instead of "see how the quality of recommendations changes"
say "see how recommendation quality changes" and maybe add on "and iteratively adjust your weights to fine-tune."